src/content/certifications/landing.json

```diff
@@ -2,5 +2,10 @@
 	"hero": {
 		"heading": "HashiCorp Cloud Engineer Certifications",
 		"description": "Each certification program tests both conceptual knowledge and real-world experience using HashiCorp multi-cloud tools. Upon passing the exam, you can easily communicate your proficiency and employers can quickly verify your results."
-	}
+	},
+	"programSummarySlugs": [
```

Comment:
❓ does this denote the order of the summaries

Reply:
💭 If that's the authoring intent, then we should add 'order' somewhere in the property name?

Reply:
i had it as `programSummaryOrder` in #1404 and then switched it to this for some reason 🤦 the `order` part does seem more important - now update to `programSummaryOrder`! 👍